index.html

```diff
@@ -41,10 +41,20 @@ <h2>Why you should use it</h2>
 			</div>
 		</div>
 		<div class="row">
+			<div class="col-sm-4 feature">
+				<h3>Modern Features</h3>
+				<p>Push notifications, link previews, new message markers, and more bring IRC to the 21st century.</p>
+			</div>
 			<div class="col-sm-4 feature">
 				<h3>Always Connected</h3>
-				<p>The Lounge never sleeps. Just sign in and pick up where you left off.</p>
+				<p>Remains connected to IRC servers while you are offline. Forget about bouncer software.</p>
+			</div>
+			<div class="col-sm-4 feature">
+				<h3>Synchronized Experience</h3>
```

Comment:
This feels off to me. "Shared sessions"? "Shared accounts"? "Seamless experience"? I'll think of others with a clearer mind.

Reply:
Actually, I would merge this section with either the previous one or the next one, as they kiiiind of play in the same area.
And that way we can re-add something about open source and/or community managed. I think the free, open source, privacy-respectful (you own your data, this is not SaaS) is a key feature and important to point it out.